examples/graph/ipynb/mpnn-molecular-graphs.ipynb

```diff
@@ -2,23 +2,20 @@
  "cells": [
```

Comment:
There's still a ton of changes in this file, there just be one line changed (the line with the typo). Revert to the initial state and fix the typo only

Reply:
Sorry, @fchollet I just changed one line on ipynb but windows or git might have added those lines due to LF to CRLF issue. Now, I reverted those changes and just changed one line on py file.